tests/app-test.js

```diff
@@ -1,10 +1,77 @@
-/* global describe, beforeEach, afterEach, it */
+/* global describe, beforeEach, afterEach */
 import { expect } from 'chai';
+import $ from 'jquery';
+import sinon from 'sinon';
+import it from './it-will';
```

Comment:
We also need a `.only` variant as that currently doesn't work with our convergent-it